openshift/e2e-common.sh

```diff
@@ -129,7 +129,35 @@ function run_e2e_tests(){
   oc -n knative-eventing set env deployment/mt-broker-controller BROKER_INJECTION_DEFAULT=true || return 1
   wait_until_pods_running $EVENTING_NAMESPACE || return 2
 
-  go_test_e2e -timeout=90m -parallel=12 ./test/e2e ./test/conformance \
+  go_test_e2e -timeout=90m -parallel=12 ./test/e2e \
+    "$run_command" \
+    -brokerclass=MTChannelBasedBroker \
+    $common_opts || failed=$?
+
+  return $failed
+}
+
+function run_conformance_tests(){
+  header "Running Conformance tests with Multi Tenant Channel Based Broker"
+  k get ns ${TEST_EVENTING_NAMESPACE} 2>/dev/null || TEST_EVENTING_NAMESPACE="knative-eventing"
```

Comment:
"k" ? That will only work on your localhost I guess?

Reply:
OK, that's in the `e2e` test function too. It's a common (installed) alias for `kubectl` - I will fix